tensorflow/python/framework/load_library.py

```diff
@@ -158,6 +158,44 @@ def load_library(library_location):
         'The file or folder to load kernel libraries from does not exist.',
         library_location)
 
+def load_pluggable_device_library(library_location):
+  """Loads a TensorFlow PluggableDevice plugin.
+  "library_location" can be a path to a specific shared object, or a folder.
+  If it is a folder, all shared objects will be loaded. when the library is
+  loaded, devices/kernels registered in the library via StreamExecutor C API
+  and Kernel/Op Registration C API are made available in TensorFlow process.
+
+  Args:
+    library_location: Path to the plugin or folder of plugins.
+      Relative or absolute filesystem path to a dynamic library file or folder.
+
+  Returns:
```

Comment:
(No need for this since there's no return value)

Reply:
done